Include needed standard C headers.

 -- Rafael Laboissiere <rafael@debian.org>  Tue, 11 Nov 2008 13:49:05 +0000

--- a/src/ANN.cpp
+++ b/src/ANN.cpp
@@ -24,6 +24,8 @@
 //		Added performance counting to annDist()
 //----------------------------------------------------------------------
 
+#include <cstdlib>
+
 #include <ANN/ANNx.h>					// all ANN includes
 #include <ANN/ANNperf.h>				// ANN performance 
 
--- ann-1.1.1.orig/src/kd_dump.cpp
+++ ann-1.1.1/src/kd_dump.cpp
@@ -33,6 +33,8 @@
 
 #include "kd_tree.h"					// kd-tree declarations
 #include "bd_tree.h"					// bd-tree declarations
+#include <cstring>
+#include <cstdlib>
 
 using namespace std;					// make std:: available
 
--- a/include/ANN/ANN.h
+++ b/include/ANN/ANN.h
@@ -89,6 +89,7 @@
 
 #include <cmath>			// math includes
 #include <iostream>			// I/O streams
+#include <cstring>
 
 //----------------------------------------------------------------------
 // Limits
--- a/ann2fig/ann2fig.cpp
+++ b/ann2fig/ann2fig.cpp
@@ -42,7 +42,8 @@
 
 #include <cstdio>						// C standard I/O
 #include <fstream>						// file I/O
-#include <string>						// string manipulation
+#include <cstring>						// string manipulation
+#include <cstdlib>
 #include <ANN/ANNx.h>					// all ANN includes
 
 using namespace std;					// make std:: accessible
